internal/commands/app/create.go

```diff
@@ -396,6 +396,7 @@ func (cmd CommandCreate) handleCreateApp(
 	return nil
 }
 
+// handleCreateTemplateApp does not take in serverless instances because they are not compatible with template apps
```

Comment:
Can we update this comment to explain what `handleCreateTemplateApp` does also. Usually one would expect comments in this format to explain the purpose of the function also